tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
```

Comment:
I would really really like to be able to iterate quickly on our list of transforms (we have already had to change the list), and I'm not sure what the downside is of loading all of the helpers in the Meteor tool. I can understand being stingy with helpers on the client, but why on the server?